Automattic/woocommerce-payments
# Add dispute details screen

Fixes #350 

#### Changes proposed in this Pull Request

Add a "Dispute Details" screen with a Challenge button linking to the evidence form:

<img width="1145" alt="Screen Shot 2020-02-10 at 1 02 55 AM" src="https://user-images.githubusercontent.com/1867547/74124684-19a0ae00-4ba1-11ea-8cdb-4fc740b588ea.png">

…linked via a new info button in Disputes list rows:

<img width="1067" alt="Screen Shot 2020-02-10 at 12 34 59 AM" src="https://user-images.githubusercontent.com/1867547/74123587-536fb580-4b9d-11ea-8cb6-d4cb06985418.png">

The copy for the details page is derived from a google doc shared by @LevinMedia.

Also:
- removes small placeholder on info button in Transactions and Disputes lists
- removes link from "Needs Response" status chip to evidence form (just added in https://github.com/Automattic/woocommerce-payments/pull/410)

Treating container width and outsize vertical padding (being discussed in https://github.com/Automattic/woocommerce-payments/pull/414) as out of scope.

The dispute info section near the top is tracked in https://github.com/Automattic/woocommerce-payments/issues/379.

The "Accept Dispute" button can also be a different PR in my view (not currently tracked in its own issue but blocking https://github.com/Automattic/woocommerce-payments/issues/376 and https://github.com/Automattic/woocommerce-payments/issues/377).

Question: should the info button only appear if the dispute needs a response? Or for all rows, as now, except we'll need to disable things / make it clear it's not actionable? (cc @LevinMedia; related: https://github.com/Automattic/woocommerce-payments/issues/379#issuecomment-583965218)

#### Testing instructions



* Open the Disputes list view, and click on the info button to the left
* Verify that appropriate text shows up for disputes with the different statuses

-------------------
